metals/src/main/scala/scala/meta/internal/metals/BuildTargets.scala

```diff
@@ -370,21 +363,6 @@ final class BuildTargets(
       .map(_.getId())
   }
 
-  private def unsafeInferBuildTarget(
```

Comment:
This method is no longer needed as now we write source-jar-name in the dependencies directory.